react-app/src/components/schemaFormUtils.js

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import { Accordion, Button, Dropdown } from 'react-bootstrap';
+import { Accordion, Button, Dropdown, Form } from 'react-bootstrap';
```

Comment:
I'm seeing a lint warning saying we don't use `Form` anymore in this file. Can it be deleletd?